spec/features/activities/project_attributes_activity_spec.rb

```diff
@@ -87,6 +87,7 @@
 
       # own fields
       expect(page).to have_selector('li', text: "Name changed from #{previous_project_attributes['name']} to #{project.name}")
+      expect(page).to have_selector('li', text: 'Project status description set (Details)')
```

Comment:
Can you move this assertion under the `expect(page).to have_selector('li', text: 'Project marked as template')` so that the order is the same for assignments and assertions?

Reply:
I sure can!